packit_service/worker/handlers/copr.py

```diff
@@ -163,6 +163,14 @@ def pre_check(self) -> bool:
             # we can't report it to end-user at this stage
             return False
 
+        if self.copr_build_helper.is_custom_copr_project_defined():
+            logger.debug(
+                "Custom Copr owner/project set. "
+                "Checking if this GitHub project can use this Copr project."
+            )
+            if not self.copr_build_helper.check_if_custom_copr_can_be_used_and_report():
+                return False
```

Comment:
```suggestion
            return self.copr_build_helper.check_if_custom_copr_can_be_used_and_report()
```

Reply:
I wanted to avoid this to not forget about the `True` value if we add another check. But am not against it.

Reply:
ok, then feel free to leave it this way, it was just a nit and your explanation makes sense